### ISSUE TITLE
Option to have tabs appear in single line

### ISSUE THREAD

**Issue by ReaderGuy42** [gitobsidiantutorial/obsidian-tabs#32]
Would it be possible to make the tabs appear in a single line, like in a browser?

Thanks :)

**Comment by ZibraMax** [comment]
That would be awesome!

**Comment by zantiu** [comment]
Yes please, the current behavior is confusing compared to a browser or IDE like VS Code

**Comment by zantiu** [comment]
Ok looks other issues request the same thing, and here is an explanation why it's still not implemented:
https://github.com/gitobsidiantutorial/obsidian-tabs/issues/16

**Comment by bonecast** [comment]
yeah this is a must and what made me stop using it. 😅

**Comment by juliang22** [comment]
^^Same, that would be amazing
